app/konnect/dev-portal/access-and-approval/sso.md

```diff
@@ -0,0 +1,129 @@
+
+1. [Use the OIDC well-known discovery endpoint](https://learn.microsoft.com/en-us/azure/active-directory-b2c/secure-api-management?tabs=app-reg-ga#get-a-token-issuer-endpoint) to find and save the `issuer` value. 
+    The `issuer` value will be used as the provider URL when configuring SSO in {{site.konnect_short_name}}.
+
+1. [Add a group claim](https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims?tabs=appui#configure-groups-optional-claims). Enable all the group types settings and select **Group ID** for each setting in the token properties type.
+
+1. [Configure an optional claim](https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims?tabs=appui#configure-optional-claims-in-your-application) with **ID** as the token type and **email** as the claim.
+
+{% endnavtab %}
+{% navtab Okta %}
+
+1. Create a [new OIDC application in Okta](https://help.okta.com/oie/en-us/content/topics/apps/apps_app_integration_wizard_oidc.htm) to manage {{site.konnect_saas}} account integration. Configure the following settings:
+    * **Application Type**: Web Application
+    * **Grant type**: Authorization Code
+    * **Sign-in redirect URIs**: `https://cloud.konghq.com/login`
+    * **Sign-out redirect URIs**: `https://cloud.konghq.com/login`
+
+    Leave this page open. You'll need the connection details here to configure your {{site.konnect_saas}} account.
+
+1. (Optional) If you want to use group claims for Konnect [developer team mappings](/konnect/dev-portal/access-and-approval/add-teams/), click the **Claims** tab in Okta to [configure a `groups` claim](https://developer.okta.com/docs/guides/customize-tokens-groups-claim/main/#add-a-groups-claim-for-a-custom-authorization-server) and fill in the following fields:
+
+    Field | Value
+    ---|---
+    Name | `groups`
+    Include in token type | ID token, Always
+    Value type | Groups
+    Filter | Select **Matches regex** from the drop-down, then enter `.*` in the field.
+    Include in | Choose **The following scopes** and select `openid`, `email`, and `profile`. 
+
+    This claim tells Okta to reference a subset of Okta groups.
+    In this case, the wildcard (`.*`) value tells Okta to make all groups
+    available for team mapping.
+
+    {:.important}
+    > If the authorization server is pulling in additional groups from
+    third-party applications (for example, Google groups), the `groups` claim
+    cannot find them. An Okta administrator needs to duplicate those groups and
+    re-create them directly in Okta. They can do this by exporting the group in
+    question in CSV format, then importing the CSV file to populate the new group.
+
+1. [Add users to the application](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-assign-apps.htm).
```

Comment:
```suggestion
1. [Add users to the Okta application](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-assign-apps.htm).
```

Reply:
Since we tab between two properties that both contain a user value we should be specific.